SU-SWS/stanford_news
# D8CORE-2675: removing the external link arrow from social media icons

# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
removing the external link arrow from social media icons

# Needed By (Date)
Immediately

# Urgency
Needed for release

# Steps to Test

1. Check out this branch
1. Clear cache
2. Verify that the arrows don't show on the social media icons for news nodes

# Affected Projects or Products
- D8core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-2675
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
